test/PrepStmt/client9.cpp

```
@@ -64,7 +64,7 @@ void run(MYSQL *mysql) {
 	uint32_t r=(uint32_t)mt_rand();
 	r=r%3000;
 	char *query=(char *)malloc(strlen(QUERY1)+16);
-	sprintf(query,QUERY1,r);
+	snprintf(query,strlen(QUERY1)+16,QUERY1,r);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)